charts/linkerd-control-plane/README.md

```diff
@@ -230,10 +230,12 @@ Kubernetes: `>=1.21.0-0`
 | proxy.image.pullPolicy | string | imagePullPolicy | Pull policy for the proxy container Docker image |
 | proxy.image.version | string | linkerdVersion | Tag for the proxy container Docker image |
 | proxy.inboundConnectTimeout | string | `"100ms"` | Maximum time allowed for the proxy to establish an inbound TCP connection |
+| proxy.inboundDiscoveryCacheUnusedTimeout | string | `""` | Maximum time allowed before an unused inbound discovery result is evicted from the cache, defaults to 90s if unspecified |
```

Comment:
The default for this Helm value is "" which causes the proxy env var to not be set, which causes the proxy to use it's own default value (e.g. 90s in this case.  Did I understand that correctly?

Would it be better to put the default here and always set the env var?  That feels more explicit than relying on an internal proxy default.  And it's more consistent with e.g. `inboundConnectTimeout` above.

Reply:
> The default for this Helm value is "" which causes the proxy env var to not be set, which causes the proxy to use it's own default value (e.g. 90s in this case. Did I understand that correctly?

That is correct, yes.

You raise a good point, I was unsure of it myself. It felt a bit easier to manage the default value if we only had it in one place (i.e. in the proxy). It does feel a bit more opaque though so maybe it's better to be more explicit.